packages/node-build-scripts/sass-compile.ts

```diff
@@ -66,5 +67,25 @@ function compileFile(inputFile: string) {
         charset: true,
     });
     fs.outputFileSync(outFile, result.css, { flag: "w" });
-    fs.outputFileSync(outputMapFile, result.map, { flag: "w" });
+    if (result.map != null) {
+        fs.outputFileSync(outputMapFile, fixSourcePathsInSourceMap({ outputMapFile, sourceMapBuffer: result.map }), {
+            flag: "w",
+        });
+    }
+}
+
+function fixSourcePathsInSourceMap({
```

Comment:
this approach seems cleaner, so I'm on board with the general direction of this PR so far 👍🏽